build.gradle

```diff
@@ -21,3 +21,6 @@ task clean(type: Delete) {
     delete rootProject.buildDir
 }
 
+task buildTestConfig(type:Exec) {
+  commandLine 'bin/build_test_config.sh'
```

Comment:
The script I'm working on will generate the test config file to `$HOME/aws-amplify/aws-sdk-android.testconfiguration.json`. Consider having two steps: one to generate the config file and one to link it, so it'll be easier to swap out the generation piece.

Reply:
@palpatim I'd consider using a `.` directory for these files to reduce the obvious litter. Sounds good otherwise!